adcircpy/forcing/winds/best_track.py

```diff
@@ -94,6 +113,26 @@ def __init__(
 
         super().__init__(nws=nws, interval_seconds=interval_seconds)
 
+    def summary(self):
+        min_storm_speed = np.min(self.speed)
+        max_storm_speed = np.max(self.speed)
+        track_length = self.track_length
+        duration = self.duration
+        min_central_pressure = np.min(self.central_pressure)
+        max_wind_speed = np.max(self.df['max_sustained_wind_speed'])
+        start_loc = (self.df['longitude'][0], self.df['latitude'][0])
+        end_loc = (self.df['longitude'].iloc[-1], self.df['latitude'].iloc[-1])
+        f = [
+            f'Summary of storm: {self.storm_id}',
+            f'min./max. track speed: {min_storm_speed} m/s, {max_storm_speed} m/s',
+            f'min. central pressure: {min_central_pressure} hPa',
+            f'max. wind speed: {max_wind_speed} kts',
+            f'Starting at: {start_loc} and ended at: {end_loc}',
+            f'Total track length: {track_length:.2f} km',
+            f'Total track duration: {duration:.2f} days',
+        ]
+        return '\n'.join(f)
```

Comment:
The client should do: `print(bt.summary())` or if you really want to use the method to print, then we should rename it to `print_summary()`, so it is named according to what it does.